CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Logic Database
 
 - Removed: It's no longer logical to push Wide Beam Blocks with Wave Beam without Wide Beam.
+- Fixed: All usages of Missiles now require the Missile Launcher.
```

Comment:
IMO it would be nice to specify which cases didnt require use of missile launcher/pb launcher. 
Because reading this, I don't actually know exactly what logic was fixed. Was all logic expected to shoot missiles without launcher? Was it only fixed for some niche usecase on hypermode? I dont know until i go check out this PR!